code/controllers/subsystem/ticker.dm

```diff
@@ -214,7 +214,11 @@ SUBSYSTEM_DEF(ticker)
 		if(GLOB.master_mode == "secret")
 			hide_mode = 1
 			if(GLOB.secret_force_mode != "secret")
-				var/datum/game_mode/smode = config.pick_mode(GLOB.secret_force_mode)
+				var/datum/game_mode/smode 
+				if(runnable_modes.len)
+					smode = config.pick_mode(GLOB.secret_force_mode)
+				else 
+					smode = new /datum/game_mode/extended()
```

Comment:
These 4 lines worked in my testing, but when I placed debug messages here they didnd't show, so someone else should probably check this

Reply:
These lines doesn't actually make the thing exclude the original mode,its just going to loop through every enabled mode until it randomly picks one it can run

Reply:
line 212 is the bit that should exclude it. When it gets the runnable modes, it checks can_start(), which now fails if there's not enough antags, instead of drafting